.env-sample

```diff
@@ -1,2 +1,2 @@
-#name of the service desk class to be instantiated. Ex: ExampleServiceDesk, TwilioFlex, GenesysServiceDesk, InContactServiceDesk
```

Comment:
could you add back `InContactServiceDesk` in the root level env-sample

Reply:
my mistake, it's back ;)